src/select/select-list.test.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+
+import { Option } from '@ampel-ui/api';
+
+import { SelectList } from '@ampel-ui/select/select-list';
+
+import { cleanup, render } from '@config/testing';
+
+const StringSelectList = SelectList as new () => SelectList<string, Option<string>>;
+
+// `scrollIntoView` is not implemented in jsdom
```

Comment:
Then how do you test it? ;)

Reply:
I don't :) Will just remove it since we won't have that feature.